### PR TITLE
Removes required attribute from reg fields that aren't always required

### PR DESCRIPTION
**Pull request by RobRuana**

### CODE REVIEW

File: uber/templates/base.html [hunk 1]
```diff
@@ -113,15 +113,6 @@
                     errorbox_y: 28
                 });
 
-                $('.jq-dte').each(function(index, element) {
```
Comment (kitsuta) [inline uber/templates/base.html]: Why was this removed?

Comment (RobRuana) [inline uber/templates/base.html]: That's something I added in an earlier pull request. The jQuery datetextentry plugin rewrites the HTML to make the original input field a hidden input field. Then it adds three new inputs for month, day, year. This code removes the `required` property from the original input field and applies it to the three newly generated inputs.

But we don't want any of them to have the `required` property, so this code is no longer useful.
